src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsCallout/ActionItem.tsx

```diff
@@ -9,7 +9,6 @@ import {
 import React, { useCallback, useEffect, useState } from 'react';
```

Comment:
Wasn't included in the scope of this PR, but can we make the margin left of the default button same as in the other places in ConditionsCallout with 8px?

![image](https://user-images.githubusercontent.com/45217314/202013218-c91edc46-b7e4-418b-b484-4b98962b8a89.png)

Reply:
again not in the scope of this PR, but when I change the visual type to Badge coloring, even if it is empty (no badge selected), the save button is still active and when I click on save nothing happens (it returns to the state before edit - element coloring). I think we should disable save when visual type selected for badge but no icon specified, or we can select a default value for icon:
![image](https://user-images.githubusercontent.com/45217314/202014405-8b66ed7c-b664-4b3b-922d-5a20a93bfe0a.png)

Reply:
again not in the scope and maybe you already fixed in in the subsequent PR, but can we make error messages red colored as we use in different places?
![image](https://user-images.githubusercontent.com/45217314/202015971-e405f111-0e40-4313-8c70-9c276e9d62fe.png)